docker-compose.yml

```diff
@@ -42,12 +41,11 @@ services:
     build:
       context: ./
       dockerfile: Dockerfile
-    command: 'yarn watch:web'
+    command: 'yarn workspace @ehpr/web start'
```

Comment:
I'm not a huge fan of the inconsistency in commands here

Reply:
FIXED

Reply:
Fixed